docker-compose.yml

```diff
@@ -51,8 +51,8 @@ services:
       - DEPLOYMENT_STAGE=dev
       - AWS_REGION=us-west-2
       - AWS_DEFAULT_REGION=us-west-2
-      - AWS_ACCESS_KEY_ID=dev_access_key_id
-      - AWS_SECRET_ACCESS_KEY=dev_secret_access_key
+      - AWS_ACCESS_KEY_ID=test
```

Comment:
Had to change these values to test so presigned urls would work. See
https://github.com/localstack/localstack/issues/3266